rest-tests/src/test/groovy/org/hawkular/metrics/rest/CassandraBackendITest.groovy

```diff
@@ -34,6 +34,66 @@ class CassandraBackendITest extends RESTTest {
   }
 
   @Test
+  void findMetricsWhenThereIsNoData() {
```

Comment:
Le 25/03/2015 13:05, jsanda a écrit :
> I spent a whole lot of time my import settings last Friday. The settings
> for static imports were not configured in the build tools repo. I think
> getting the settings aligned is still a bit a work in progress.

The settings JAR is a work in a progress, the convention isn't :)

static java.*
blank line
static javax.*
blank line
static javax.*
blank line
static javax.*
blank line
static javax.*